Platform/ApollolakeBoardPkg/BoardConfig.py

```diff
@@ -209,14 +209,14 @@ def GetImageLayout (self):
 		img_list  = []
 		img_list.extend ([
 					# Padding to ensure all other components in OBB partition will be aligned at 4KB boundary
-					# 0xB80 assumes 7 files in BpdtIBB (IBBP.man, BPM.met) + (IPAD, IBBL, IBBM,  OBB, CFGDATA)
-					# 0xF40 assumes 5 files in BpdtOBB (OPAD,  PLD,  VAR, MRCD, PROV)
+					# 0xB00 assumes (IBBP.man, BPM.met) + (IPAD, IBBL, IBBM,  OBB, FWUP, CFGD, PLD, VAR, MRCD) in BpdtIBB
+					# 0x180 assumes (OPAD,  PROV, EPLD) in BpdtOBB 
```

Comment:
How is the actual padding size calculated based on the number of files?

Reply:
Unfortunately there is no clear published document  to know that since the alignment is arranged internally by the FITC tool.  Current approach is to try FITC stitching, and then check the layout alignment in the final IFWI.  From the offset, we will know the gap to pad.  Using this information we can decide the IPAD/OPAD size.   
This is only used by FITC stitching.  For open source stitching, the layout is fully controlled by the LoaderStitch.py script.  IPAD/OPAD will not be used at all.